class.jetpack.php

```diff
@@ -476,6 +483,11 @@ private function __construct() {
 			Jetpack_Heartbeat::init();
 		}
 
+		if ( Jetpack::is_active() && defined( 'REST_REQUEST' ) ) {
```

Comment:
I was using4.5 But I didn't update the code here to move the register to a safe place yet. I've just labeled this PR to "In Progress" as there's a lot to be changed.